presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -272,10 +268,6 @@ private static IntegerStatistics toIntegerStatistics(DwrfProto.IntegerStatistics
 
     private static DoubleStatistics toDoubleStatistics(DwrfProto.DoubleStatistics doubleStatistics)
     {
-        if (!doubleStatistics.hasMinimum() && !doubleStatistics.hasMaximum()) {
```

Comment:
Double stats are now allowed to have null min/max here. But I see in the following lines they are still not allowed to be NaN values -- will this cause different hash result?

Reply:
We won't write stats when there is a NaN (see `DoubleStatisticsBuilder`) so this is ok